packages/core/apps/core-apps-server-internal/tsconfig.json

```diff
@@ -33,6 +33,7 @@
     "@kbn/core-ui-settings-server",
     "@kbn/monaco",
     "@kbn/core-http-server-internal",
+    "@kbn/core-http-router-server-internal",
```

Comment:
It probably doesn't have any impact, but it's annoying that we have to add a new dependency just for testing, and that we can't separate the runtime ones from the testing ones.

Reply:
One alternative is to mock out the kibanaResponseFactory based on what we know the handler needs, but I thought this is a bit cleaner as it assumes less about how the handler works. If you have a clear preference about I can remove it and mock out a minimal response factory here instead?

Reply:
No preference TBH, you can go ahead!

I wonder if @elastic/kibana-operations has any open issues / thoughts on ways to improve this.